modules/deploy/pages/deployment-option/cloud/managed-connectors/transforms/smt-cast.adoc

```diff
@@ -0,0 +1,34 @@
+= Cast
+:description: How to set Cast Single Message Transform for a connector.
```

Comment:
The descriptions for all the SMTs do not provide much value. Could we get it to describe shortly what the SMT actually do? This way it is easier to read the index page and navigate to the interesting one.